mitodl/open-discussions
# remove redundant profile image, move dot

#### What are the relevant tickets?

closes #761 

#### What's this PR do?

This removes the `<UserInfo />` component that we had before in the sidebar. I also moved the 'profile incomplete' dot up to the new user menu.

#### How should this be manually tested?

Make sure that everything with the nav sidebar works well. Also, set `SETTINGS.profile_complete` to `false`, and make sure a little red dot shows up near your profile image (in the top right).
